### ISSUE TITLE
How to make "selectedItemRenderer" not removable

### ISSUE THREAD

**Issue by SouroucheB** [kenshoo/react-multi-select#147]
Hello there,

Here is my code :
```
<MultiSelect
    id="multi-select"
    name="multi-select"
    showSelectAll
    selectedItemRenderer={this.renderSelectedItem}
/>
```
And :
```
renderSelectedItem = ({ item, height }) => {
    const { label, ...customProps } = item;
    return <p {...customProps}>{label}</p>;
};
```

And my question : By doing this, the selected item on the right column remains removable while it is exactly the behavior i'm trying to avoid. While digging a little, it seems that it is the container that triggers the action of suppression.

Could someone help me please to understand what I'm missing here ?

Thanks.

**Comment by stepovat** [comment]
hi @SouroucheB, 
you are right, the container handles item removal

We are actually right now working on feature to support locked items on a right side, in case if they are passed with field "disabled: true" in the list of selectedItems property.

Do you think it will fit your goals?
If not, could you please provide more details of what you are planning to have?

Thanks

**Comment by SouroucheB** [comment]
Hi @stepovat, thanks for your reponse,
It will indeed fit one of my goal  because right now, when I put something disabled in the left column, it still also remains clickable, which is seems to be an unexpected behavior, I guess.

But I feel like my final goal is a little like this one. I would love to have the ability to replace all the targeted element with the "renderers" you provide like `itemRenderer` or `selectedItemRenderer`. What's means that I would replace the inner container but also the most higher level container which trigger the mouse event. I am clear ?

Thanks

**Comment by stepovat** [comment]
@SouroucheB 
1. Regarding what you said about putting disabled in left column and it is still clickable - that doesn't sound good, could you please provide an example of your code with inputs?
Just in case, we have a working example of disabled items in left list [here](https://kenshoo.github.io/react-multi-select/?knob-Show%20search=true&knob-Show%20select%20all=true&knob-Height%20%25=50&selectedKind=React%20Multi%20Select&selectedStory=With%20some%20of%20the%20items%20disabled&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

2. Regarding ability to override render for wrapper container - it's an interesting suggestion and we are always welcoming changes propositions. 
If selected locked items (that we are currently working on) won't fit your needs - we will be really glad to see PR with your suggested changes!

Thanks

**Comment by stepovat** [comment]
Hi @SouroucheB, 
just wanted to notify, that locked items support was released in v1.1.3
Please tell if any questions appear,
Thanks

**Comment by liorheber** [comment]
@SouroucheB closing this issue. Feel free to re-open if needed.
Also please note that you can always block the click event from the wrapper by stopping the event from bubbling up to the parent in your renderer.

I hope your issue is solved.
